src/components/MyKiva/EarnedBadgesSection.vue

```diff
@@ -134,6 +142,21 @@ const getBadgeDate = badge => {
 const loadMoreBadges = () => {
 	visibleOffset.value += 1;
 };
+
+const clickBadge = badge => {
+	const badgeCategory = badge.level === 0
```

Comment:
Since we're also passing the `level` to the analytics, I think we could utilize `badge.challengeName`, if you wanted to simplify this logic.